sdcm/sct_events/setup.py

```diff
@@ -55,6 +56,9 @@ def start_events_device(log_dir: Optional[Union[str, Path]] = None,
     time.sleep(EVENTS_SUBSCRIBERS_START_DELAY)
 
     # Default filters.
+    workload_line = 'workload prioritization - update_service_levels_from_distributed_data: an error occurred while ' \
+                    'retrieving configuration'
```

Comment:
Personally, I don't like such things in the code: defining a new variable just because of formatting problems: there is no reuse of this constant anywhere. As for me, it's better to change the call of `EventsSeverityChangerFilter` below.

Reply:
removed this variable, and re-formatted the call to `EventsSeverityChangerFilter`